bin/utils/ngx_handle.lua

```diff
@@ -8,25 +8,19 @@ local function create_dirs(necessary_dirs)
     end
 end
 
-local function remove_ngx_conf(ngx_conf)
-    os.command("mv -f "..ngx_conf .. " /tmp/")
-end
 
 local function ngx_command(args)
     if not args then
         error("error args to execute nginx command.")
         os.exit(1)
     end
 
-    local orange_conf, prefix, ngx_conf, ngx_signal = "", "", "", ""
```

Comment:
does this line of code need to be deleted? After making changes to the code, please submit it after local verification.